examples/directml/stable_diffusion/user_script.py

```diff
@@ -148,22 +160,30 @@ def unet_load(model_name):
     return model
 
 
-def unet_conversion_inputs(model):
-    return tuple(unet_inputs(1, torch.float32).values())
+def unet_conversion_inputs_512(model):
```

Comment:
We can avoid multiple functions by storing the image size in a global variable, and `stable_diffusion.py` can change the default value of that to 512 or 768 depending on the model requirements.